### PR TITLE
Detail popup window

### PR DESCRIPTION
**Pull request by hopeforkoami**
in this branch, implement the following interactions:

    the popup with details about the project appears when the user clicks the button to check project details, 
    the close (X) button, the popup disappears when the user clicks 
I used a JavaScript array to store all of the information for all projects.
I updated the main page so that the projects section is created dynamically using the information stored in that JavaScript object.
I  implemented the popup window.
I implemented the popup windows for desktop or mobile

### CODE REVIEW

File: js/portofolio.js [hunk 1]
```diff
@@ -1,7 +1,104 @@
+const dataSource = [
```
Comment (smunozmo) [inline js/portofolio.js:1]: - [ ] You are almost there! However, for some reason the projects section is hidden when the site is loaded, therefore I was not able to review it:

Screen Shot 2022-09-08 at 11 32 11 PM

Let's fix that bug to get it to display this section.

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @hopeforkoami, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/portofolio.js (hunk 1 above)
Comment (hopeforkoami) [reply js/portofolio.js:1]: thank you 
please would you mind checking it again
i forgot a last push

Comment (hopeforkoami) [reply js/portofolio.js:1]: Hi @smunozmo, 
thank you
please would you mind checking it again
i forgot a last push but i just push it

File: js/portofolio.js [hunk 2]
```diff
@@ -12,6 +109,91 @@ function close() {
...
+            + '<div class="modal_image"  onclick="closepopup()"> </div>'
+            + '<div class="desktop_title">'
+              + '<h2 class="modal_title">'}${dataSource[index].card_popup_title}</h2>`
+            + '<div class="buttons for_desktop">'
+              + `<a href="${dataSource[index].card_live_link}"><button class="left_button modal_button">See live <img src="assets/see_live_icon.svg" alt="see live icon"> </button></a>`
+              + `<a href="${dataSource[index].card_source_link}"><button class="left_button modal_button">See Source <img src="assets/github_icon.svg" alt="github icon"> </button></a>`
+            + '</div></div><ul class="modal_popup_technologies">';
+  dataSource[index].card_technologies.forEach((element) => {
+    modalcontent += `<li class="work_language_item">${element}</li>`;
+  });
```
Comment (thecodechaser) [inline js/portofolio.js:131]: - [ ] For the desktop screen, Please make sure the image in the model display in full size, and your design should match with the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design.

### Your design
image

### Figma design
image

File: js/portofolio.js [hunk 3]
```diff
@@ -12,6 +109,91 @@ function close() {
...
+  modalcontent += `</ul><p class="popup_description">${dataSource[index].card_description}</p><div class="buttons for_mobile">`
+                + `<a href="${dataSource[index].card_live_link}"><button class="left_button modal_button">See live <img src="assets/see_live_icon.svg" alt="see live icon"> </button></a>`
+                + `<a href="${dataSource[index].card_source_link}"><button class="left_button modal_button">See Source <img src="assets/github_icon.svg" alt="github icon"> </button></a>`
```
Comment (thecodechaser) [inline js/portofolio.js:134]: - [ ] Please remove the black border around the model buttons and add some border radius to match the Figma design.

### Your design
image

### Figma design
image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @hopeforkoami, and Team 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @hopeforkoami, @Miliyonayalew, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**